Resources/Prototypes/Reagents/pyrotechnic.yml

```
@@ -30,10 +30,21 @@
     Poison:
       effects:
       - !type:HealthChange
+        conditions:
+        - !type:OrganType # Goobstation - Yowie
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)